src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -112,6 +154,10 @@ const FlatTableRow = React.forwardRef(
                     firstColumnExpandable
                       ? handleCellKeyDown
                       : undefined,
+                  cellIndex: index,
+                  reportCellWidth:
+                    index < rowHeaderIndex ? reportCellWidth : undefined,
```

Comment:
I think that we can as well

```suggestion
                    index < rowHeaderIndex && reportCellWidth
```

Reply:
I think this would actually pass a `boolean` to the prop rather than a `function/ undefined`

Reply:
It will/should pass the function anyway

Reply:
It passes a boolean with your suggestion if the first condition fails, I need it to either pass a function or undefined. If you update the code to your suggestion you get a prop type warning

![image](https://user-images.githubusercontent.com/44157880/112816495-61bab280-9079-11eb-8549-bba17cae4081.png)